### PR TITLE
ioctl 1.2.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by nandahkrishna** [comment]
This seems to be the updated `homepage`: https://docs.iotex.io/reference/ioctl-cli-reference/installation

**Comment by carlocab** [comment]
Is this a stable release?

**Review by raullenchai — DISMISSED** [review]

File: Formula/ioctl.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ioctl < Formula
   desc "Command-line interface for interacting with the IoTeX blockchain"
-  homepage "https://docs.iotex.io/developer/get-started/ioctl-install.html"
-  url "https://github.com/iotexproject/iotex-core/archive/v1.1.3.tar.gz"
-  sha256 "1a9e50a5831d9543489187944b1fc73bdaf3ae87c9cbe8f46810f25a915d0e24"
+  homepage "https://docs.iotex.io/"
+  url "https://github.com/iotexproject/iotex-core/archive/v1.2.0.tar.gz"
+  sha256 "4f71a36cde2e57d2d75fab8c37a89dd711af6539efe1ed847783464dcbfeeb54"
```
Comment (dustinxie) [inline Formula/ioctl.rb:5]: we had officially released v1.2.0
pls double-check the sha256 is correct

Comment (dustinxie) [reply Formula/ioctl.rb:5]: thanks for homebrew team actively checking on our status 💯

Comment (nandahkrishna) [reply Formula/ioctl.rb:5]: Thank you! Seems like the `sha256` is correct: `4f71a36cde2e57d2d75fab8c37a89dd711af6539efe1ed847783464dcbfeeb54`.

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**